perun/postprocess/regression_analysis/extensions/plot_models.py

```diff
@@ -0,0 +1,147 @@
+""" Extension for regression model coefficients transformation into array of points. The points
+    array can be then used for model plotting as a series of lines forming a curved line.
+"""
+
+import perun.postprocess.regression_analysis.tools as tools
+import numpy as np
+
+
+# Default model curve smoothness specified as number of points generated from x interval
+# The higher, the smoother are the curves
+DEFAULT_SMOOTHNESS = 51
+
+
+def model_plot_computation(data):
+    """ The model plotting computation wrapper. Handles common operations regardless of model,
+        uses 'y_pts_func' to compute y values, which can be model-specific
+
+    Expects 'y_pts_func', 'x_interval_start', 'x_interval_end', 'b0' and 'b1' keys in the
```

Comment:
y_pts_func is f(x) that is used to compute the y value, right? Maybe we could name it just "f_x" or something like this, or is it used somewhere already?